sdk/translation/Azure.AI.Translation.Text/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Please provide details here.

Reply:
Since this is the initial version, I recommend adding a brief description of the initial set of supported features under the "Features Added" section.

Reply:
Added "Features Added" section.